modules/ROOT/pages/design-migrating-ms.adoc

```diff
@@ -0,0 +1,132 @@
+For example, when you are designing an API specification in API Designer, you can simulate a call to an endpoint’s GET method to see whether the response body for the 200 status code is as you want it. As a further example, if you are evaluating whether you want to implement an API specification that is in Exchange, you can simulate a call to an endpoint’s GET method to find out what the response body looks like.
+
+
+== How to Migrate an API-Specification Project to Version 2 of the Mocking Service
+
+If the mocking service was switched on for an API-specification project before January 10, 2019 and has been running continuously since then, follow these steps to migrate the project to version 2 of the mocking service:
+
+. In the notification that appears in the top-right corner after you open the project, click *Upgrade*.
++
+*Result:* The mocking service is switched off and then on. The form of the base URI that the mocking service generates now differs from the form of the previous base URI.
++
+Version 1 of the mocking service used base URIs of the form `+https://mocksvc.mulesoft.com/mocks/{ID}+`.
++
+Version 2 uses base URIs of the form `+https://anypoint.mulesoft.com/mocking/api/v1/links/{new-ID}+`.
++
+For example, the URI for a GET method for an endpoint named `/customers` in a particular specification might have looked like this with version 1 of the mocking service:
++
+`+https://mocksvc.mulesoft.com/mocks/7d19e624-a0d6-4d77-9fb8-8c78a8aeb575/api/customers+`
++
+Version 2 of the mocking service changes the URI to this and uses a different ID:
++
+`+https://anypoint.mulesoft.com/mocking/api/v1/links/62a7d6af-ebc7-43fa-a815-e415e329f2ff/api/customers+`
+
+. If any third-party tools are testing endpoints in the API specification for that project, replace the previous base URI with the new base URI in the links that those tools are using.
++
+For example, suppose that this URI is being used in a tool:
++
+`+https://mocksvc.mulesoft.com/mocks/7d19e624-a0d6-4d77-9fb8-8c78a8aeb575/api/customers+`
++
+You would need to replace `+https://mocksvc.mulesoft.com/mocks/7d19e624-a0d6-4d77-9fb8-8c78a8aeb575+` with `+https://anypoint.mulesoft.com/mocking/api/v1/links/{new-ID}+`, where `new-ID` is the ID that version 2 of the mocking service generates.
++
+You can get the new base URI from version 2 of the mocking service after step 1 above.
+
+== New Features in Version 2 That You Can Use in API Designer
+
+Version 2 of the mocking service, which was released on January 10, 2019, introduced the following features that you can use in API Designer.
+
+=== Support for OAS 2.0
+
+You can now test endpoints that are in API specifications that are written in OAS 2.0.
+
+
+=== Behavioral Headers
+
+Add headers to your simulated calls to methods, changing the behavior of the mocking service in response to those calls The following new headers are available. See xref::apid-behavioral-headers.adoc[Add Behavioral Headers to Simulated API Calls] for full details.
```

Comment:
```suggestion
...in response to those calls. The following new headers are available:  
```

Reply:
Add period, as shown above, and I suggest moving the cross-ref to follow the list rather than come between "the following..." and the actual list.